webknossos/webknossos/dataset/sampling_modes.py

```diff
@@ -0,0 +1,4 @@
+class SamplingModes:
```

Comment:
enum?

Reply:
I thought so too, not sure why it wasn't before.